### PR TITLE
Chapter1 homework 9

### PR DESCRIPTION
**Pull request by mkasarci**

### CODE REVIEW

File: Src/BootCamp.Chapter/BinaryConverter.cs [hunk 1]
```diff
@@ -8,12 +9,58 @@ public static class BinaryConverter
...
+                number +=Char.GetNumericValue(binaryNumbers[i]) * power;
+            }
+
+            return long.Parse(number.ToString());
         }
 
         public static string ToBinary(long number)
         {
-            return "";
+            string binaryValue = "";
```
Comment (jdphenix) [inline Src/BootCamp.Chapter/BinaryConverter.cs]: Consider using a `StringBuilder` and returning `StringBuilder.ToString()` at the end.

File: Src/BootCamp.Chapter/BinaryConverter.cs [hunk 2]
```diff
@@ -8,12 +9,58 @@ public static class BinaryConverter
...
+                    number /= 2;
+                }
+
+                if (number == 0)
+                    return binaryValue;
+
+                binaryValue = "0" + binaryValue;
+                number /= 2;
+            }
+            //return Convert.ToString(Convert.ToInt32(number, 16), 2);
```
Comment (jdphenix) [inline Src/BootCamp.Chapter/BinaryConverter.cs]: Remove unnecessary comment.

File: Src/BootCamp.Chapter/ArrowMovement.cs [hunk 3]
```diff
@@ -18,7 +18,31 @@ public static class ArrowMovement
...
+                    return '\u21A7';
+                case 's':
+                    return '\u21A7';
+                case 'C':
+                    return '\u21a5';
+                case 'c':
+                    return '\u21a5';
+            }
+
+            return 'a';
```
Comment (jdphenix) [inline Src/BootCamp.Chapter/ArrowMovement.cs]: Looks pretty good. Don't forget the default case should return ↥.

**Review by jdphenix — CHANGES_REQUESTED** [review]
Looks pretty good, just a couple of nitpicks.

**Comment by mkasarci** [comment]
I've changed the code and use StringBuilder. Can you please check again?

**Review by jdphenix — APPROVED** [review]
Good work, thank you!

**Pull request closed without merging**